<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="MSI" failures="0" tests="5" name="Default test" time="0.174" errors="0" timestamp="2022-09-22T17:06:57 IST">
  <testcase classname="tests.BaseTest" name="@BeforeClass setup" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.util.Properties.getProperty(String)&amp;quot; because &amp;quot;this.configProperties&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.Properties.getProperty(String)" because "this.configProperties" is null
at tests.BaseTest.setup(BaseTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="tests.GenerateLoad" name="validateRequest" time="0.0">
    <skipped/>
  </testcase> <!-- validateRequest -->
  <testcase classname="tests.GenerateLoad" name="validateRequest" time="0.0">
    <skipped/>
  </testcase> <!-- validateRequest -->
  <testcase classname="tests.GenerateLoad" name="validateRequest" time="0.0">
    <skipped/>
  </testcase> <!-- validateRequest -->
  <testcase classname="tests.GenerateLoad" name="validateRequest" time="0.0">
    <skipped/>
  </testcase> <!-- validateRequest -->
  <testcase classname="tests.GenerateLoad" name="validateRequest" time="0.0">
    <skipped/>
  </testcase> <!-- validateRequest -->
  <testcase classname="tests.BaseTest" name="@BeforeSuite preSetup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite preSetup -->
</testsuite> <!-- Default test -->
